registries/intranet_agents.hocon

```diff
@@ -16,6 +16,16 @@
 {
     include "registries/aaosa.hocon"
 
+ 
+    "metadata": {
+        "description": "Multi-agent system that mimics the intranet of a major corporation. Allows you to interact with and get information from various departments such as IT, Finance, Legal, HR, and Procurement. The top-level agent receives a question and, in coordination with down-chain agents, provides comprehensive answers.",
+        "tags": ["AAOSA", "industry"],
```

Comment:
I don't think `Industry` is needed here as all these agent networks will move to an `Industry` folder in #408

Reply:
Maybe `Intranet` would be useful